snuba/datasets/schemas/table_schemas.py

```diff
@@ -203,6 +217,9 @@ def __get_local_source_table_name(self) -> str:
     def __get_local_destination_table_name(self) -> str:
         return self._make_test_table(self.__local_destination_table_name)
 
+    def _get_local_engine(self) -> str:
+        pass
```

Comment:
This seems a bit like an inheritance smell. If there's really no better way to do this (I didn't look in depth) it should probably be commented.

Should maybe the engine only be a parameter to the writable schemas?